README.md

```diff
@@ -222,24 +285,6 @@ However, wrapping GDAL has a few drawbacks for Python developers and data scient
 
 With the introduction of projects like Numba, Python gained new ways to provide high-performance code directly in Python, without depending on or being constrained by separate C/C++ extensions. `xarray-spatial` implements algorithms using Numba and Dask, making all of its source code available as pure Python without any "black box" barriers that obscure what is going on and prevent full optimization. Projects can make use of the functionality provided by `xarray-spatial` where available, while still using GDAL where required for other tasks.
 
-#### Contributors
-
-- @brendancol
-- @thuydotm
-- @jbednar
-- @pablomakepath
-- @kristinepetrosyan
-- @sjsrey
-- @giancastro
-- @ocefpaf
-- @rsignell-usgs
-- @marcozimmermannpm
-- @jthetzel
-- @chase-dwelle
-- @SAN154
-- @SapirLastimoza-Dooley
-- @lex-c
-
-Cite our code:
+**Cite our code:**
```

Comment:
Shouldn't this be a headline?

`#### Cite our code`

Reply:
Thanks Timo! Changed as requested.